reformulator.py

```diff
@@ -999,5 +994,8 @@ def load_db(self) -> Dict:
             whi(f"Launching reformulator.py with args '{args}' and kwargs '{kwargs}'")
             AnkiReformulator(*args, **kwargs)
             sync_anki()
-    except Exception:
+    except AssertionError as e:
+        red(e)
+    except Exception as e:
+        red(e)
```

Comment:
Are you sure about this?

Reply:
This is a personal thing, we can change it.

Basically, I make it so that if we fail an assert, it's probably an issue with the invocation, and we log it. If it's another type of exception, then we probably don't expect it, so we log it, but we also re-raise the exception to print the stack trace. I can remove it, but it's helping me with debugging.

Reply:
Whatever helps you helps all of us here so no biggy